modules/navier_stokes/test/tests/ins/RZ_cone/tests

```diff
@@ -167,4 +175,27 @@
     issues = '#14901'
     requirement = 'The system shall compute an accurate Jacobian using automatic differentiation when solving the incompressible Navier Stokes equations in an axisymmetric coordinate system with SUPG and PSPG stabilization'
   [../]
+  [ad_rz_cone_by_parts_traction_steady_supg_pspg]
+    type = 'Exodiff'
+    input = 'ad_rz_cone_by_parts_traction_steady_stabilized.i'
+    exodiff = 'ad_rz_cone_by_parts_traction_steady_stabilized_out.e'
+    issues = '#14901'
+    requirement = 'The system shall be able to solve the steady incompressible Navier-Stokes equationswith SUPG and PSPG stabilization and a first order velocity basis using an automatic differentiation, vector variable implementation in an RZ coordinate system while integrating the pressure term by parts, using a traction form for the viscous term, and applying a natural outflow boundary condition.'
+  []
+  [rz_cone_by_parts_traction_steady_supg_pspg]
+    type = 'Exodiff'
+    input = 'rz_cone_by_parts_traction_steady_stabilized.i'
+    exodiff = 'rz_cone_by_parts_traction_steady_stabilized_out.e'
+    issues = '#14901'
+    requirement = 'The system shall be able to solve the steady incompressible Navier-Stokes equationswith SUPG and PSPG stabilization and a first order velocity basis using a hand-coded Jacobian, standard variable implementation in an RZ coordinate system while integrating the pressure term by parts, using a traction form for the viscous term, and applying a natural outflow boundary condition and reproduce the results of the AD, vector variable implementation.'
+    custom_cmp = 'rz-cone-with-globals.cmp' # To ignore the vector variable components automatically generated by libMesh/ParaView
+  []
+  [ad_rz_cone_by_parts_traction_steady_supg_pspg_jac]
+    type = 'PetscJacobianTester'
+    difference_tol = 1e-6
```

Comment:
It is magical

Reply:
So nice not having to worry about Jacobians when doing complex vector operations and stabilization methods! Development is so fast!